src/test/unit/mocks/onePublicationJson.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
we already have a publication json file called `returnedArtefacts.json`, if you only need 1 you just need to `[0]` the data you parse